ocaml/xapi/quicktest_raw_vdi_import.ml

```diff
@@ -0,0 +1,60 @@
+(*
+ * Copyright (C) 2013 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+(* Check the VDI.copy support for copying disk deltas *)
+
+open Sparse_encoding
+open Client
+open Quicktest_common
+open Http
+
+let http request f =
+  let open Xmlrpc_client in
+  let transport =
+    if !using_unix_domain_socket
+    then Unix Xapi_globs.unix_domain_socket
+    else SSL(SSL.make ~use_fork_exec_helper:false (), !host, 443) in
+  with_transport transport (with_http request f)
+
+let import_raw_vdi ~session_id ~task_id f =
+  (*let uri = Printf.sprintf "/import_raw_vdi?session_id=%s&vdi=%s&task_id=%si&chunked=true"
```

Comment:
Thanks. since it was the original way to call the http handler with a uri, which not fits here.
I will remove them.